p2p/conn/connection_bench_test.go

```diff
@@ -566,3 +608,114 @@ func BenchmarkMConnection_Multiple_ChannelID(b *testing.B) {
 		})
 	}
 }
+
+// BenchmarkMConnection_Multiple_ChannelID assesses the max bw/send rate
```

Comment:
[nit] comment and function name should match

```suggestion
// TestMConnection_Message_Order_ChannelID assesses the max bw/send rate
```

Reply:
thanks, fixed https://github.com/celestiaorg/celestia-core/pull/1193/commits/a4b979aaaf31cd6f3b93eb081075e195a630ac2f